### PR TITLE
Add hidaya v1.0 theme

### PR DESCRIPTION
**Pull request by Stunkymonkey**

### CODE REVIEW

File: export_themes.xml [hunk 1]
```diff
@@ -34,6 +34,24 @@
       <section>movies</section>
     </supports>
   </theme>
+  <theme>
+    <name>Hidaya Theme</name>
+    <identifier>hidaya</identifier>
+    <description>beautiful theme</description>
```
Comment (bugwelle) [inline export_themes.xml]: Could you add a more descriptive text here? e.g. `dark theme with search functionality`?

File: export_themes.xml [hunk 2]
```diff
@@ -34,6 +34,24 @@
...
+    <description lang="de">Schönes Theme</description>
```
Comment (bugwelle) [inline export_themes.xml]: Eine etwas bessere Beschreibung wäre noch schöner. :-)

Grüße von 'nem Heidelberger nach Stuttgart. 😃

**Review by bugwelle — CHANGES_REQUESTED** [review]
I'll look at your code in a minute. :-)

Could you please adapt the description? Besides that: 👍

File: export_themes.xml (hunk 2 above)
Comment (Stunkymonkey) [reply export_themes.xml]: sorry war einfach copy&paste aus der `metadata.xml`. Sollte ich auch da noch anpassen :see_no_evil:

Comment (bugwelle) [reply export_themes.xml]: Alles gut, dafür sind Reviews ja da. 😄

Comment (Stunkymonkey) [reply export_themes.xml]: Grüße zurück :smile:

**Comment by bugwelle** [comment]
Please see https://github.com/Stunkymonkey/hidaya/pull/2

Looks good 👍

**Comment by Stunkymonkey** [comment]
maybe it makes sense to update to a new /updated release

**Comment by bugwelle** [comment]
> maybe it makes sense to update to a new /updated release

What do you mean? You can make a new release (and increase `<version>`). Or delete the old tag and create the new one (and update the checksum of this PR). :-)

**Review by bugwelle — APPROVED** [review]

**Comment by bugwelle** [comment]
Screen Shot 2020-10-24 at 19 04 41


Works like a charm! Thank you very much for this theme. I've exported my media files and it works well!

For future changes, please do not update the git tag / GitHub release but instead create a new version and adapt it here. :-)

**Comment by Stunkymonkey** [comment]
Sure otherwise the shasum will break. It was only done while the PR was not merged yet.

**Comment by bugwelle** [comment]
> Sure otherwise the shasum will break. It was only done while the PR was not merged yet.

👍

**Pull request merged**